src/test/regress/expected/prepare.out

```diff
@@ -184,6 +184,17 @@ SELECT name, statement, parameter_types, result_types FROM pg_prepared_statement
       |     UPDATE tenk1 SET stringu1 = $2 WHERE unique1 = $1;           |                                                    | 
 (6 rows)
 
+CREATE TABLE MANFRED_22834 (id int primary key, name varchar(20));
```

Comment:
nit - not sure if we should use "MANFRED" in open-source. let's pick a different name

Reply:
I agree here. Let's rename this test to something else.

Reply:
ok